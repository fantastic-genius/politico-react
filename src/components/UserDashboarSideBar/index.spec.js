import React from 'react';
import { shallow } from 'enzyme';
import UserDashboardSideBar from './index';

describe('<UserDashboardSideBar />', () => {
  describe('Rendering', () => {
    it('should render UserDashboardSideBar correctly in debug mode', () => {
        const component = shallow(<UserDashboardSideBar />);
        expect(component).toMatchSnapshot();
    });
  })

})

