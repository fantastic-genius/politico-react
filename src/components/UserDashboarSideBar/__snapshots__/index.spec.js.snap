// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserDashboardSideBar /> Rendering should render UserDashboardSideBar correctly in debug mode 1`] = `
<aside>
  <Connect(UserInfo) />
  <div
    className="menu"
  >
    <ul>
      <li>
        <Link
          to="/dashboard"
        >
          Dashboard
        </Link>
      </li>
      <li>
        <Link
          to="/vote"
        >
          Vote
        </Link>
      </li>
      <li>
        <a
          href="view_parties.html"
        >
          Parties
        </a>
      </li>
      <li>
        <a
          href="my_votes.html"
        >
          My Votes
        </a>
      </li>
      <li>
        <a
          href="petition.html"
        >
          Submit Petition
        </a>
      </li>
      <li
        className="treeview"
      >
        <a
          href="#"
        >
          <span>
            Settings
          </span>
          <span
            className="to-right"
          >
            <i
              className="arrow down"
            />
          </span>
        </a>
        <ul
          className="treeview-menu hide"
        >
          <li>
            <a
              href="profile.html"
            >
              Update Profile
            </a>
          </li>
          <li>
            <a
              href="change_password.html"
            >
              Change Password
            </a>
          </li>
        </ul>
      </li>
    </ul>
  </div>
</aside>
`;
